src/absil/il.fs

```diff
@@ -8,14 +8,18 @@ module (*internal*) Microsoft.FSharp.Compiler.AbstractIL.IL
 
 
 open Internal.Utilities
+#if FABLE_COMPILER
```

Comment:
It may be a good idea to retain the conditional as there is some type overwriting that's only needed when compiled with Fable.